website/docs/ref/macro.md

```diff
@@ -4,6 +4,10 @@
 
 ## Installation
 
+:::tip
+For those who prefer not to use Babel, Lingui offers the [SWC Plugin](/docs/ref/swc-plugin.md) as an alternative for `@lingui/macro`.
+:::
+
```

Comment:
I think that may confuse people, in the way they would install only swc-plugin instead of @lingui/macro, but in fact they should install both of them (@lingui/macro for types and imports) and swc-plugin for actual compilation

What do you think about it?

Reply:
@thekip hm, what about saying just `For those who prefer not to use Babel, Lingui offers the SWC Plugin as an alternative`?

Reply:
That sounds correct